rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -326,10 +326,11 @@ public Wallet getNoSpendWalletForLiveFederations() throws IOException {
      * @param btcTxSerialized The raw BTC tx
      * @param height The height of the BTC block that contains the tx
      * @param pmtSerialized The raw partial Merkle tree
+     * @param logSuccessfulLock Whether to log an event if the lock succeeds
      * @throws BlockStoreException
      * @throws IOException
      */
-    public void registerBtcTransaction(Transaction rskTx, byte[] btcTxSerialized, int height, byte[] pmtSerialized) throws IOException, BlockStoreException {
+    public void registerBtcTransaction(Transaction rskTx, byte[] btcTxSerialized, int height, byte[] pmtSerialized, boolean logSuccessfulLock) throws IOException, BlockStoreException {
```

Comment:
Given that the `BridgeSupport` already has the `RskSystemProperties` and the _executionBlock_, wouldn't be possible to just use that configuration?